x/metadata/keeper/keeper.go

```diff
@@ -209,24 +204,10 @@ func (k Keeper) checkAuthzForMissing(
 			grantee := types.MustAccAddressFromBech32(signer)
 
 			for _, msgType := range msgTypeURLs {
-				authorization, exp := k.authzKeeper.GetCleanAuthorization(ctx, grantee, granter, msgType)
```

Comment:
These changes do not look like they should be reverted here.

Reply:
yeah looks we have a bad merge from main :( .. reverted all metadata files to match main